examples/gpt2.py

```diff
@@ -34,7 +34,7 @@ def __init__(self, dim, n_heads, linear=Linear):
     self.dim = dim
     self.head_dim = dim // n_heads
 
-  def __call__(self, x:Tensor, cache_k:Optional[Tensor], cache_v:Optional[Tensor], start_pos:int, mask:Optional[Tensor]) -> Tensor:
+  def __call__(self, x:Tensor, cache_k:Optional[Tensor], cache_v:Optional[Tensor], start_pos:int, mask:Optional[Tensor], jit_ctx:Optional[Dict[Variable,int]]=None) -> Tensor:
```

Comment:
How is jit_ctx used in this function?

Reply:
in jitted call, the underlying variable values are replaced by `jit_ctx` https://github.com/tinygrad/tinygrad/blob/89cd380bfc34ceb7a535b2ec4d6083f95778cdc0/tinygrad/jit.py#L31